packages/kbn-config-schema/src/types/nullable_type.test.ts

```diff
@@ -24,16 +24,36 @@ test('returns value if specified', () => {
   expect(type.validate('test')).toEqual('test');
 });
 
-test('returns null if null', () => {
+test('returns null if null for string', () => {
   const type = schema.nullable(schema.string());
   expect(type.validate(null)).toEqual(null);
 });
 
-test('returns null if undefined', () => {
+test('returns null if null for number', () => {
+  const type = schema.nullable(schema.number());
+  expect(type.validate(null)).toEqual(null);
+});
+
+test('returns null if null for boolean', () => {
+  const type = schema.nullable(schema.boolean());
+  expect(type.validate(null)).toEqual(null);
```

Comment:
**optional** shouldn't we prefer to use `toBe` for js primitives? `toEqual` performs equal check recursively

Reply:
changed to use `toBe()` for primitive tests